I created a function that plots emission prob.

a-dacko opened the pull request:
hiii

emmekeaarts committed:
fixed bug

Commit:
I created a function that plots emission prob.

emmekeaarts reviewed (changes requested):
For the file to be a part of the package, the file can only contain (and should start with) help file documentation and the function itself (so no loading the package and setting values for objects outside of the function)

a-dacko committed:
Here I created the plot.mHMM_emiss function. I added the documentation. It plots the emission probabilities distribution by states.
Soon I'll add a new variable that is going to indicate different ways of displaying plots.

a-dacko commented:
I added documentation to my plot functions. The things that still has to be done are:
-fix the x-axes labelling
-add a mode variable to switch between step-wise and dependent variable wise view
-if ggplot loaded use it to plot the graphs
-I wanted to add two rows of legends because with larger number of depending variables it could be not very plausible to view. I tried to do that  but something is wrong with my loops so I'' need to find a way.

a-dacko committed:
Added the simulating data part to the tutorial of the mHMMbayes package

a-dacko committed:
typo error

a-dacko committed:
Merge branch 'emmekeaarts:master' into NewOla

emmekeaarts reviewed (approved):
Looks great! you can do another pull request for the updates still to come to improve the basic version we now have.
